example/Transaction/example-authorize-retry.php

```diff
@@ -0,0 +1,67 @@
+<?php declare(strict_types=1);
+
+use Ticketpark\SaferpayJson\Request\Exception\SaferpayErrorException;
+use Ticketpark\SaferpayJson\Request\RequestConfig;
+use Ticketpark\SaferpayJson\Request\Transaction\AuthorizeRequest;
+
+require_once __DIR__ . '/../../vendor/autoload.php';
+require_once __DIR__ . '/../credentials.php';
+
+// A token you received after initializing a transaction (see example-initialize.php)
+
+$token = 'xxx';
+
+// The request ID you received after the first authorize request fails with a Behavior of RETRY or RETRY_LATER
+
+$requestId = 'your_request_id';
```

Comment:
> The request ID you received after the first authorize request fails

If I understand correctly, this is is not really the case. The request id must be provided by the merchant, even with the first request, if they have the intention to use the retry functionality. At least that's what I get from the docs:

> The RequestId is meant to identify this one request. Not the whole payment, only this one request. It has to be generated by the merchant-system. Please make sure, that it is unique for each different request.
https://docs.saferpay.com/home/integration-guide/licences-and-interfaces/error-handling#the-requestid-and-retryindicator

Or do I misunderstand something here?

Reply:
You are right, the request id identifies the single request. You create this unique ID automatically with the first request when you create the request header and no existing ID is passed.  

From my point of view, I would continue to have this generated by the request header on the first attempt, since the response in the response header returns the original requestID. This can then be explicitly set to this one on the next attempt. In this case, I saw no advantage in explicitly creating this ID, since this is currently being done anyway as it is currently built in here https://github.com/Ticketpark/SaferpayJsonApi/blob/d60e2a2c6b63310647608f5a9cc2ec957ac9f8ab/lib/SaferpayJson/Request/Container/RequestHeader.php#L42-L44

But maybe the comment can be described a little more clearly or do you have other concerns?

Reply:
Oh great, the library is doing more than I remembered 😅
Then it's maybe really just a matter of adjusting the comment.